bitbots_head_behavior/src/bitbots_head_behavior/head_node.py

```diff
@@ -25,7 +25,7 @@ def run(dsd):
 
     :returns: Never
     """
-    rate = Rate(125)
+    rate = Rate(60)
```

Comment:
why reduce the rate? higher rates mean smoother motion -> less forces acting on the robot -> less falling

Reply:
But the head behavior doesn't include any interpolation and just sets the final goals. In certain conditions, it checks the path for collisions and this check gets very CPU heavy at such high rates. The dsd also doesn't include many sequences which might take multiple steps.

Reply:
I also talked with Timon about this.

Reply:
ah okay then it makes sense. I thought that we interpolate